### ISSUE TITLE
Training-related issues

### ISSUE THREAD

**Issue by Delete12137** [toharl/soft#3]
Thank you very much for your work！
“We first pre-train the student with SLS for 6 epochs, and then add the uncertainty module for another 80 epochs with a learning rate of 0.0001 and a batch size of 64. "
How to understand this sentence？How do you do this to train the model？The source code has run, but there is a big gap between the performance and the results of the paper。
![image](https://user-images.githubusercontent.com/105090442/222879854-363c63c2-d85a-406b-9640-b21f58c7c181.png)
